awx/main/dispatch/periodic.py

```diff
@@ -21,6 +22,9 @@ def run_continuously(self):
         def run():
             ppid = os.getppid()
             logger.warning('periodic beat started')
+
+            set_connection_name('periodic')  # set application_name to distinguish from other dispatcher processes
```

Comment:
I should note - I don't ever see this. I believe that's because it _quickly_ closes the connection after sending off periodic tasks. Nonetheless, I believe it's working correctly, the service just cleans up after itself. This is a good thing.